File: fun-challenges/py/fizzbuzz.py
def fzbz():
    for i in list(range(1,101)):
        if not i % 15:
            print('fizzbuzz')
        elif not i % 5:
            print('buzz')
        elif not i % 3:
            print('fizz')
        else:
            print(i)


fzbz()

File: fun-challenges/cs/Fizzbuzz.cs
using System;

  class FizzBuzz
  {
        static void Main()
        {
            for (int i = 1; i < 101; i++)
            {

            bool FIZZ = i % 3 == 0;
            bool BUZZ = i % 5 == 0;
            bool FIZZBUZZ = i % 15 == 0;
            string fizz = "Fizz";
            string buzz = "Buzz";
            string fizzbuzz = "FizzBuzz";

                if (FIZZBUZZ)
                {
                    Console.WriteLine(fizzbuzz);
                }
                else if(BUZZ)
                {
                    Console.WriteLine(buzz);
                }
                else if(FIZZ)
                {
                    Console.WriteLine(fizz);
                }
                else
                {
                    Console.WriteLine(i);
                }
            }
        }
    }

